### PR TITLE
Adding desription for  built-in MDATP admin role

### PR DESCRIPTION
**Pull request by ojrb**
This PR solver the issue  #3375 , I add the description for built-in MDATP admin role in a note, as requested, @mypil please review and Merge !

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-atp/user-roles-windows-defender-advanced-threat-protection.md [hunk 1]
```diff
@@ -59,6 +59,10 @@ The following steps guide you on how to create roles in Windows Defender Securit
 After creating roles, you'll need to create a machine group and provide access to the machine group by assigning it to a role that you just created. 
 
 
+>[!NOTE]
+>The Windows Defender ATP administrator (default) role has administrator permissions. The administrator permissions cannot be assigned to any other role. On groups assigned the Windows Defender ATP administrator role have access to all machine groups.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-atp/user-roles-windows-defender-advanced-threat-protection.md]:
```suggestion
>The Windows Defender ATP administrator (default) role has administrator permissions. The administrator permissions cannot be assigned to any other role. On groups assigned, the Windows Defender ATP administrator role has access to all machine groups.
```

File: windows/security/threat-protection/windows-defender-atp/user-roles-windows-defender-advanced-threat-protection.md [hunk 2]
```diff
@@ -76,6 +80,7 @@ After creating roles, you'll need to create a machine group and provide access t
 2.	Click the drop-down button and select **Delete role**.
 
 
-##Related topic
+
+## Related topic
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-atp/user-roles-windows-defender-advanced-threat-protection.md]:
```suggestion
## Related topics
```

**Comment by mypil** [comment]
@ojrb  - please merge the commit on the copy/edit review made by JohanFreelancer9 / nenonix  or make the necessary changes on the PR following the suggestions made. Thank you.

File: windows/security/threat-protection/windows-defender-atp/user-roles-windows-defender-advanced-threat-protection.md (hunk 1 above)
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-atp/user-roles-windows-defender-advanced-threat-protection.md]:
```suggestion
>The Windows Defender ATP administrator (default) role has administrator permissions. Administrator permissions cannot be assigned to any other role. Only those who are (?) assigned the Windows Defender ATP administrator role have access to all machine groups.
```

**Comment by mypil** [comment]
@ojrb - kindly check on the comments from JohanFreelancer9 & nenonix. Thanks.

**Comment by mypil** [comment]
@ojrb - I would like to follow up on this. Kindly check on the comments from JohanFreelancer9 & nenonix. Thanks.

**Comment by ojrb** [comment]
Hi @mypil, I commited the changes proposed for @JohanFreelancer9  but can not merge the changes for @nenonix as you can see the branch has conflict, I don't know why ? Can you help on this in order to merge the Nicole changes ?

**Comment by nenonix** [comment]
@ojrb please check your fork for the conflict. Proposed edits would not cause conflict, but your fork being out of date with the main repo will do that,

**Comment by MamaDoc** [comment]
All systems are 2 months old.   She has set up for Mac and android.   I am
lenex & window iPad IPHONE.  All my passwords have been changed

**Comment by mypil** [comment]
@ojrb - kindly follow the recommendation from nenonix to check for conflicts. You can use this [guide](https://help.github.com/en/articles/addressing-merge-conflicts) as reference.  Thanks.

**Comment by ojrb** [comment]
Hi @nenonix and @mypil I review this conflict and the problem is that in Master the file  **windows/security/threat-protection/windows-defender-atp/user-roles-windows-defender-advanced-threat-protection.md** does not exist, was deleted. When I try to access the original file on the issue (Content Source), I receive the error 404 page not exist! This is the problem that in Master the file not exist and the PR can not be executed! Please review with the Document owner what's happens with the file in the Master !

**Comment by mypil** [comment]
@mjcaparas - Can you please assist to check, approve and merge the content of this PR based on issue #3375?

Thank you.

**Comment by mjcaparas** [comment]
Pls resolve conflict

**Comment by mypil** [comment]
@mjcaparas - I validated and confirmed the findings of @ojrb where the content source seems to have a 404 error. This may be the cause of the conflict and we need your guidance on how that can be resolved. Thank you.

**Comment by mjcaparas** [comment]
In line 85 and 86 - update the file names to:
basic-permissions.md
machine-groups.md

File: windows/security/threat-protection/windows-defender-atp/user-roles-windows-defender-advanced-threat-protection.md [hunk 3]
```diff
@@ -59,6 +59,10 @@ The following steps guide you on how to create roles in Windows Defender Securit
 After creating roles, you'll need to create a machine group and provide access to the machine group by assigning it to a role that you just created. 
 
 
+>[!NOTE]
+>The Windows Defender ATP administrator (default) role has administrator permissions. Administrator permissions cannot be assigned to any other role. Only those who are assigned the Windows Defender ATP administrator role, has access to all machine groups.
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-atp/user-roles-windows-defender-advanced-threat-protection.md]:
```suggestion
>The Windows Defender ATP administrator (default) role has administrator permissions. Administrator permissions cannot be assigned to any other role. Only those who are assigned the Windows Defender ATP administrator role have access to all machine groups.
```

File: windows/security/threat-protection/windows-defender-atp/user-roles-windows-defender-advanced-threat-protection.md [hunk 4]
```diff
@@ -76,6 +80,7 @@ After creating roles, you'll need to create a machine group and provide access t
 2.	Click the drop-down button and select **Delete role**.
 
 
-##Related topic
+
+## Related topics
 - [User basic permissions to access the portal](basic-permissions-windows-defender-advanced-threat-protection.md)
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-atp/user-roles-windows-defender-advanced-threat-protection.md]:
```suggestion
- [User basic permissions to access the portal](basic-permissions.md)
```

File: windows/security/threat-protection/windows-defender-atp/user-roles-windows-defender-advanced-threat-protection.md [hunk 5]
```diff
@@ -76,6 +80,7 @@ After creating roles, you'll need to create a machine group and provide access t
...
-- [Create and manage machine groups](machine-groups-windows-defender-advanced-threat-protection.md)
\ No newline at end of file
+- [Create and manage machine groups](machine-groups-windows-defender-advanced-threat-protection.md)
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-atp/user-roles-windows-defender-advanced-threat-protection.md]:
```suggestion
- [Create and manage machine groups](machine-groups.md)
```

Comment (mjcaparas) [reply windows/security/threat-protection/windows-defender-atp/user-roles-windows-defender-advanced-threat-protection.md]: line 85 doesn't look to have been updated

**Comment by ojrb** [comment]
I accept the @nenonix changes ! but still the conflict !

**Comment by ojrb** [comment]
Ready to merge ! I rename the file to user-roles.md !

**Pull request merged**